src/models/utils.py

```diff
@@ -1,44 +1,81 @@
+# pylint: disable=possibly-used-before-assignment
+import os
+import sys
+
 import torch
 import torchvision.transforms.functional as F
 from facenet_pytorch import InceptionResnetV1
-from .pdt import PDT
-import os
-from torch import Tensor
 from onnx2torch import convert
-import sys
+from torch import Tensor
+
+from .pdt import PDT
 
 
 def find_threshold(distance_metric):
+    """
+    Returns the threshold value based on the specified distance metric.
+
+    Args:
+        distance_metric (str): The distance metric to use. Can be 'euclidean', 'euclidean_l2', or 'cosine'. # pylint: disable=line-too-long
+
+    Returns:
+        float: The threshold value corresponding to the distance metric.
+
+    Raises:
+        ValueError: If the distance metric is not one of 'euclidean', 'euclidean_l2', or 'cosine'.
+    """
     if distance_metric == "euclidean":
         return 1.1
     if distance_metric == "euclidean_l2":
         return 1.1
-    elif distance_metric == "cosine":
+    if distance_metric == "cosine":
```

Comment:
Not sure if it's a breaking change to change the name from euclidean to another name to make it more clear that it is the L1 norm, but if it isn't a breaking change, I would save change the name to "l1_norm" or something similar